pytential/qbx/__init__.py

```diff
@@ -603,6 +603,9 @@ def exec_compute_potential_insn_fmm(self, actx: PyOpenCLArrayContext,
                     geo_data.tree().user_source_ids,
                     insn.kernel_arguments, evaluate))
 
+        from sumpy.fmm import SumpyTranslationClassesData
+        translation_classes_data = SumpyTranslationClassesData(actx.queue,
+                geo_data.traversal())
         tree_indep = self._tree_indep_data_for_wrangler(
```

Comment:
Would it make sense to compute the translation classes as part of the wrangler creation?

Reply:
Wrangler doesn't have access to a `queue`. Should I add that?

Reply:
You can just make a private, temporary queue:
```python
with cl.CommandQueue(ctx) as queue:
   ...
```